codegen/lib/graphql_java_gen.rb

```diff
@@ -274,4 +274,34 @@ def abstract_types
       end
     end
   end
+
+  def java_doc(element)
+    doc = ''
+    unless element.description.nil?
+      description = wrap_text(element.description, 100)
```

Comment:
So is this a concern with Shopify's schema?  I don't actually know.

Since I think we could implement this properly if needed, since the output of markdown is meant to be line wrapped.  However, I would prefer to avoid doing it on the input.  For instance, doing the line wrapping here could cause list items to get broken up into multiple lines that don't get treated as a single list item.